webpush/encrypt.go

```diff
@@ -55,12 +55,36 @@ import (
 	"strings"
 )
 
+// Encryption version.
+type Version int
+
+const (
+	// The version that is widely deployed with WebPush (as of 2016-11).
+	AESGCM = iota
```

Comment:
I think as far as I know, `iota` is a way to declare numeric constants from a starting value (default 0) which increments by 1. So in this case, `AESGCM` = 0, and `AES128GCM` = 1.